module_utils/network_lsr/argument_validator.py

```diff
@@ -11,6 +11,9 @@
 from ansible.module_utils.network_lsr.utils import Util  # noqa:E501
 
 UINT32_MAX = 0xFFFFFFFF
+# a MAC address for type ethernet requires 6 octets
+# a MAC address for type infiniband requires 20 octets
+MAC_ADDR_OCTETS = [6, 20]
```

Comment:
This is wrong.

The kernel has no limitation minimum MAC address length.
And the maximum is not 20, but in `/usr/include/linux/netdevice.h`:

#define MAX_ADDR_LEN	32		/* Largest hardware address length */

That is 32 octets and 64 hex string characters.

Reply:
The reason that I define `MAC_ADDR_OCTETS = [6, 20]` is because I saw the validation check here,
https://github.com/linux-system-roles/network/blob/main/module_utils/network_lsr/argument_validator.py#L1483

And in NM,it was also clear indicated,
```
     * ---ifcfg-rh---
     * property: mac-address
     * variable: HWADDR
     * description: IBoIP 20-byte hardware address of the device (in traditional
     *    hex-digits-and-colons notation).
     *    Note that for initscripts this is the current MAC address of the device as found
     *    during ifup. For NetworkManager this is the permanent MAC address. Or in case no
     *    permanent MAC address exists, the MAC address initially configured on the device.
     * example: HWADDR=01:02:03:04:05:06:07:08:09:0A:01:02:03:04:05:06:07:08:09:11
     * ---end---
     
     /* ---keyfile---
     * property: mac-address
     * format: usual hex-digits-and-colons notation
     * description: MAC address in traditional hex-digits-and-colons notation
     *   (e.g. 00:22:68:12:79:A2), or semicolon separated list of 6 bytes (obsolete)
     *   (e.g. 0;34;104;18;121;162)
     * ---end---
```